### PR TITLE
Update yotta configuration parameters

### PR DESCRIPTION
**Pull request by artokin**
-Add more parameters to yotta configuration
-Remove parameter pskd from init as pskd is now in yotta config
-Enable Sleepy end device configuration via yotta config

### CODE REVIEW

**Comment by artokin** [comment]
@mikter, @SeppoTakalo @terhei would you please review?

File: source/thread_tasklet.c [hunk 1]
```diff
@@ -200,36 +213,86 @@ void thread_tasklet_parse_network_event(arm_event_s *event)
...
+        return;
+    }
+
+    // PSKd
+    if (sizeof(YOTTA_CFG_MBED_MESH_API_THREAD_PSKD) < 6) {
+        tr_error("PSKd length must be > 6: %s", YOTTA_CFG_MBED_MESH_API_THREAD_PSKD);
+        return;
+    }
+
+    char *dyn_buf = ns_dyn_mem_alloc(sizeof(YOTTA_CFG_MBED_MESH_API_THREAD_PSKD));
```
Comment (SeppoTakalo) [inline source/thread_tasklet.c]: Why do you use ns_dynmem in mbed OS?

**Comment by SeppoTakalo** [comment]
Get rid of the ns dynmem on mbed OS modules. It forces you to depend that module. Use malloc() and free() in mbed OS when not in interrupt context.

**Comment by artokin** [comment]
nsdynmem is initialized by this module and we can't this remove dependency.

**Comment by terhei** [comment]
+1

**Comment by SeppoTakalo** [comment]
OK, but then, just initialize it and use the malloc/free for your own purposes. Leave the nsdynmenlib for the nannostack only.

**Pull request merged**